### PR TITLE
feat: it's a bird, it's a plane, it's vortex-all!

### PR DESCRIPTION
**Pull request by danking**
One crate to rule them all and in the darkness bind them.

### CODE REVIEW

File: vortex-rs/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+[package]
+name = "vortex-rs"
+description = "Vortex with all builtin codecs and a sampling compressor."
```
Comment (AdamGS) [inline vortex-rs/Cargo.toml]: Do we want this crate to re-export all the underlying dependencies?

Comment (AdamGS) [reply vortex-rs/Cargo.toml]: We should probably also move a bunch of the docs in `vortex-array` here, but that can be a separate PR

**Comment by robert3005** [comment]
@danking as @AdamGS points out you have to reexport dependencies, otherwise none of the code referenced is accessible and it's just a dead dependency

**Comment by AdamGS** [comment]
and probably with some module structure, the same way `arrow` has `arrow::schema` and the `arrow_schema` crate

**Comment by danking** [comment]
I did this:
```
pub use vortex::*;
pub use {
    vortex_alp as alp, vortex_buffer as buffer, vortex_bytebool as bytebool,
    vortex_datetime_dtype as datetime_dtype, vortex_datetime_parts as datetime_parts,
    vortex_dict as dict, vortex_dtype as dtype, vortex_error as error, vortex_expr as expr,
    vortex_fastlanes as fastlanes, vortex_flatbuffers as flatbuffers, vortex_fsst as fsst,
    vortex_proto as proto, vortex_roaring as roaring, vortex_runend as runend,
    vortex_runend_bool as runend_bool, vortex_sampling_compressor as sampling_compressor,
    vortex_scalar as scalar, vortex_schema as schema, vortex_serde as serde,
    vortex_zigzag as zigzag,
};
```

**Comment by danking** [comment]
You can see what it looks like in bench-vortex and pyvortex now.

**Comment by danking** [comment]
I think we liked `vortex` as the module name and `vortex-all` as the package

**Comment by danking** [comment]
Auto merge disabled until https://github.com/spiraldb/vortex/pull/1124 merges.

**Comment by danking** [comment]
@robert3005 Do you have more mega PRs to land?

**Comment by robert3005** [comment]
None that you need to block on

**Review by robert3005 — APPROVED** [review]

**Pull request merged**